src/OrleansTelemetryConsumers.Counters/OrleansPerfCounterTelemetryConsumer.cs

```diff
@@ -31,7 +31,7 @@ public OrleansPerfCounterTelemetryConsumer()
             this.isInitialized = new Lazy<bool>(this.Initialize, true);
             if (!AreWindowsPerfCountersAvailable())
             {
-                logger.Warn(ErrorCode.PerfCounterNotFound, "Windows perf counters not found -- defaulting to in-memory counters. " + ExplainHowToCreateOrleansPerfCounters);
+                LoggerExtensions.Warn(logger, ErrorCode.PerfCounterNotFound, "Windows perf counters not found -- defaulting to in-memory counters. " + ExplainHowToCreateOrleansPerfCounters);
```

Comment:
Not sure I understand why the extension method can't still be used. Was this change by design?

Reply:
the ILoggerExtension.cs I introduced brought compiling ambitious to OrleansTelemetryConsumer proj, hence this change to call the extension method in a more explicit way. Maybe you know how to fix this?

Reply:
Not sure I follow. I downloaded your branch to try this, and just reverting the changes to this file still compiles

Reply:
hmm on my machine vs 17 cannot find the exact method. It cannot decide ILogger.Warn or Logger.Warn to use.... 

 maybe I try again,